LICENSE

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2012-2021 Fabien Potencier
```

Comment:
what about `Fabien Potencier, Dariusz Rumiński`? ;-)

Reply:
That is valid too, either one is fine, It was on 2 lines before I got here, so I kept on 2 lines